pkg/config/config.go

```diff
@@ -1,16 +1,22 @@
 package config
 
-import "os"
+import (
+	"os"
+	"sync"
+)
 
 var urldbConn string
+var m = &sync.Mutex{}
 
 func SetURLDBConn(urlconnection string) {
 	urldbConn = urlconnection
 }
 
 func GetURLDBConn() string {
+	m.Lock()
 	if urldbConn == "" {
 		urldbConn = os.Getenv("URL_DB_CONN")
```

Comment:
If we want to set the variable from the environment variable only once during the lifetime of the application,
I would place this code inside the `init` function of this package.
https://stackoverflow.com/a/24790378/4196220

Reply:
That's a good idea, Alain. Yeah, I know about init() function and even read exactly that post on stackoverflow before, but since I we do explicit assignment in start function I decided to leave these functions as they are. We cannot use init function with arguments, that's the problem, but in start function we can and do

Reply:
@alaingilbert